src/ontology/uberon-edit.obo

```diff
@@ -224708,6 +224686,47 @@ relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 relationship: part_of UBERON:0007365 ! rumen
 property_value: dcterms-date 2023-02-08T08:53:48Z xsd:dateTime
 
+[Term]
+id: UBERON:8600000
+name: lobular bronchiole
+def: "A preterminal bronchiole that branches into terminal bronchioles." [PMID:16543587, PMID:27611890]
```

Comment:
If "preterminal bronchiole" is the same as a 'lobular bronchiole' as noted with the exact synonym, consider revising to remove this tautology.

Reply:
Thank you for your comment. I have replaced 'preterminal bronchiole' with 'conductive airway'.